packages/msgraph/test/Adaptor.test.js

```diff
@@ -38,67 +45,286 @@ describe('execute', () => {
   });
 });
 
-describe('create', () => {
-  it.skip('makes a post request to the right endpoint', async () => {
+describe('getSites', () => {
```

Comment:
These unit tests are excellent, well done. They help me understand the APIs and their expected usage as well!